channeldb/kek.txt

```diff
@@ -0,0 +1,3241 @@
+=== RUN   TestInvoiceAddTimeSeries
```

Comment:
accidental add? :)

Reply:
lol yeh should be gone in the latest version